src/parser/druid/feral/CHANGELOG.js

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 
-import { Abelito75, Zeboot, LeoZhekov, Tora, Xcessiv } from 'CONTRIBUTORS';
+import { Abelito75, Zeboot, LeoZhekov, Tora, Xcessiv, Tiboonn } from 'CONTRIBUTORS';
 import { change, date } from 'common/changelog';
 
 export default [
+  change(date(2021, 1, 16), <>Added spell information for conduits</>, Tiboonn),
```

Comment:
these fragments too and then it should be good to go.

In general, if its just text, just wrap it with ' '. If it has something like SpellLink or something like that, then use <> </>

Reply:
makes sense!